src/main/java/com/klarna/hiverunner/HiveRunnerCore.java

```diff
@@ -64,7 +64,7 @@ private HiveShellContainer buildShell(List<? extends Script> scripts, Object tes
     hiveShellBuilder.setCommandShellEmulation(config.getCommandShellEmulator());
 
     HiveShellField shellSetter = loadScriptUnderTest(testCase, hiveShellBuilder);
-    if (scripts != null) {
+    if (!scripts.isEmpty()) {
```

Comment:
Can you remove this `if` as I think passing an empty list through should be fine now since there is no `null` coming in.

Reply:
Surely if the list is empty we don't want to override the scripts right? If I take it out, it fails:(

Reply:
I meant removing just the `if` check and always running the code that is now in the block. If the list is empty I think where it gets used will handle that fine...

Reply:
OK, I understand now, we only want to override the scripts if there is actually something in this list, otherwise we want to leave them "as is".